crates/rust-analyzer/tests/rust-analyzer/support.rs

```diff
@@ -47,11 +40,6 @@ impl<'a> Project<'a> {
         self
     }
 
-    pub(crate) fn with_sysroot(mut self, yes: bool) -> Project<'a> {
```

Comment:
oh my, did we lost sysroot loading skipping somewhere?

I'll investigate, thank you so much for noticing this!

bors r+

Reply:
Yeah, around https://github.com/rust-analyzer/rust-analyzer/pull/7177, IIRC.